test.js

```diff
@@ -1,5 +1,12 @@
 var t = "hello";
 
-if (t === "hello') {
+var selec = '[data-dikkemik';
+
+
+if (t === 'hello') {
     console.log('ok');
 }
+
+if (true) {
+  var sjaak_de_man = 'false';
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'sjaak_de_man' is not in camel case.](https://www.codacy.com/app/rubstar/rvs-pr-testing/file/2472821147/issues/source?bid=3260976&fileBranchId=3260991#l11)